Utils/build/lib/dataset/dataset.py

```diff
@@ -0,0 +1,74 @@
+from collections import namedtuple
+from typing import List
+
+annotation = namedtuple('annotation', 'tag, start, end, ne')
+
+
+class Dataset:
+    def __init__(self):
+        self.dataset = []  # храним набор tuple (text, [(annotation, start, end, ne)])
+        self.crop_sentences = []
+
+    def load_sentences(self, ann_file_path: str, text_file_path: str):
+        with open(text_file_path, 'r') as text_file:
+            text = text_file.read()
+
+        with open(ann_file_path, 'r') as ann_file:
+            annotations = []
+            for line in ann_file.readlines():
+                tokens = line.split()
+                annotations.append(annotation(tokens[1], tokens[2], tokens[3], tokens[4]))
+
+        cur_tuple = (text, annotations)
+        self.dataset.append(cur_tuple)
+
+    def _get_left(self, local_text: str, ann: annotation, window_size) -> List[str]:
```

Comment:
Add please a documentation to each method:

`def foo(a, b):`

`"""`
`Do smth`
`param :a is..`
`param :b is..`
`return: `
`"""`